.github/workflows/workers.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy
+
+on:
```

Comment:
We can add a paths filter to reduce this workflow running times: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet